Sources/Account/Account.swift

```diff
@@ -229,7 +285,7 @@ extension OwnedTxOut {
         } else {
             spentBlock = nil
         }
-
+        
```

Comment:
Extra whitespace

Reply:
I checked all the whitespace, I think this might just be a visual thing with github

Reply:
It's definitely there, line 288, but there's other whitespace in the repo, so I can just remove it in a separate PR.

Reply:
Err, actually almost all the whitespace I'm noticing is being added by this PR. There's about 12 instances of trailing whitespace being added by this PR.

But it's a CBB so I wouldn't worry about it right now.